layer_metadata_restorer_test.go

```diff
@@ -40,8 +40,7 @@ func testLayerMetadataRestorer(t *testing.T, when spec.G, it spec.S) {
 		layerDir, err = ioutil.TempDir("", "lifecycle-layer-dir")
 		h.AssertNil(t, err)
 		logger := &log.Logger{Handler: &discard.Handler{}}
-		platform := platform.NewPlatform("0.7")
-		layerAnalyzer = lifecycle.NewLayerMetadataRestorer(logger, layerDir, platform, skipLayers)
+		layerAnalyzer = lifecycle.NewLayerMetadataRestorer(logger, layerDir, skipLayers)
```

Comment:
Just to make sure I'm not missing something: we're planning to delete `restorer_platform06_test.go` and `restorer_platform07_test.go`, right?

Reply:
Yeah that'd be next.

Reply:
There's also this issue which is related: https://github.com/buildpacks/lifecycle/issues/587